### ISSUE TITLE
`allowCompression` invalid?

### ISSUE THREAD

**Issue by angcyo** [miguelpruivo/flutter_file_picker#1472]
`allowCompression` is not used in the source code?

**Comment by miguelpruivo** [comment]
Yes it is, you can see an example [here](https://github.com/miguelpruivo/flutter_file_picker/blob/04e32daa8180e4d93a96c1d0ffba9d61feb25b10/ios/Classes/FilePickerPlugin.m#L136).
